### PR TITLE
Updated credentials, discovery and ecommerce Dockerfiles

### PR DESCRIPTION
**Pull request by clintonb**
- Added file to allow bower to install as root
- Added xvfb, firefox, and gettext packages for testing and local development

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce @edx/devops the tests for the Catalog Service used to finish in about 7-8 minutes. Since moving to testing on Docker, they now take about 15 minutes. Much of that time is due to the tests being slower (perhaps because of host file system?); however, we can start chipping away at the run time by installing these packages in the container rather than every time we run a test.

File: docker/build/ecommerce/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,16 @@
+# docker build -f docker/build/ecommerce/Dockerfile . -t edxops/ecommerce
```
Comment (rlucioni) [inline docker/build/ecommerce/Dockerfile:1]: Did you leave this here intentionally?

**Review by rlucioni — APPROVED** [review]
Looks good. Will you be uploading updated images when this merges?

File: docker/build/ecommerce/Dockerfile (hunk 1 above)
Comment (clintonb) [reply docker/build/ecommerce/Dockerfile:1]: Yes, for convenience.

**Comment by clintonb** [comment]
@rlucioni already pushed.

**Comment by feanil** [comment]
https://github.com/edx/ecommerce/pull/1026#discussion_r89206094

**Comment by feanil** [comment]
@clintonb I feel like you have forgotten this conversation ^

**Pull request merged**